### ISSUE TITLE
Option to not lowercase keys

### ISSUE THREAD

**Issue by technosophos** [jofas/env_file_reader#1]
Thanks for your crate. We're just now starting to use it for [one of our projects](https://github.com/deislabs/wagi/).

We have to work with some environment variables whose key names are case-sensitive (e.g. it is possible to have `FOO=bar` and `Foo=baz`). Would it be possible to have an option to not lowercase all keys by default? I'm happy to write up a PR if you have a preferred way of implementing it. But I didn't want to break compatibility for anyone else.

**Comment by bacongobbler** [comment]
For reference, this is acceptable input:

```
><> export foo=1
><> export Foo=2
><> export FOO=3
><> env | grep -i foo
FOO=3
foo=1
Foo=2
```

**Comment by jofas** [comment]
Hi, 

without being too overtly emotional over the fact that someone is actually using one of my crates, thanks for the issue.

I've just released `v0.2.0` which treats variable names as case-sensitive. I honestly just lower-cased them to spare my eyes. 
Variable names can now also contain hyphens and I made the `read_file` and `read_files` path arguments more generic (`&str` to `impl AsRef<Path>`), to make the interface better align to the standard library.

I hope that fixes your problem.
